.github/workflows/ansible-lint.yml

```diff
@@ -0,0 +1,15 @@
+name: ansible-lint
+
+on:
+  pull_request:
+
+jobs:
+  build:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install Nix
+        uses: cachix/install-nix-action@v22
+      - name: Run ansible-lint
+        run: nix develop --command bash -c "cd ansible && ansible-lint"
```

Comment:
Cool!

The `nix develop` is probably going to be pretty slow since it has to build an entire nix store each go. It might be worth tacking on something like https://github.com/DeterminateSystems/magic-nix-cache-action/ to get better caching behaviour.

Reply:
Thanks for the suggestion, just added it